library/core/src/ops/try_trait.rs

```diff
@@ -0,0 +1,225 @@
+use crate::ops::ControlFlow;
+
+/// The trait used for a variety of operations related to short-circuits,
+/// such as the `?` operator, `try {}` blocks, and `try_*` methods.
```

Comment:
Not in love with this description of the Try trait but having trouble thinking of anything better. I normally describe the try trait as "modeling control flow, specifically early returns", but I'm not confident this will get the point across more effectively than your current iteration. Interested in your thoughts.

Reply:
Yeah, I'm not sure either.  Based on <https://doc.rust-lang.org/nightly/std/ops/index.html#traits> I've updated the first line to just talk about the "operators" that use this, and have added some more text before the example.

Reply:
that seems reasonable, I never noticed the ops traits were all described that way.